packages/app/src/services/subgraph.ts

```diff
@@ -77,14 +77,14 @@ interface RolesQueryResponse {
         executionOptions: string
         clearance: ConditionType
         functions: {
-          functionSig: string
+          sighash: string
           executionOptions: string
           wildcarded: boolean
           parameters: {
-            parameterIndex: number
-            parameterType: ParameterType
-            parameterComparison: ParamComparison
-            parameterComparisonValue: string
+            index: number
+            type: ParameterType
+            comparison: ParamComparison
+            comparisonValue: string
```

Comment:
Actually: Shouldn't this rather be `string[]`, considering the GraphQL schema type is `[Bytes!]!`?

Reply:
Yes, I think you are right. Thanks!